open-metadata-implementation/user-interfaces/ui-admin-services/docs/user/README.md

```diff
@@ -0,0 +1,112 @@
+are defined in a JSON **[configuration document](../concepts/configuration-document.md)**.
+When the configuration document is loaded to the OMAG server platform, the logical OMAG server
+is started, and the capabilities defined in the configuration document are activated.
+
+In an open metadata landscape, it is anticipated that there may be multiple
+instances of the logical OMAG Server running, each performing a different role.
+The capabilities of each of these instances would be defined in a different configuration document.
+They could all, however, be loaded into the same OMAG server platform, or distributed across
+different OMAG server platforms.
+
+The configuration document for a specific logical OMAG server is identified by the server's name.
+This is passed on the URL of every admin services API request along with the user
+id of the administrator.  By default, the configuration is stored in a file called:
+
+```
+omag.server.{serverName}.config
+```
+
+The administration services that set up this file all begin with a URL like this:
+
+```
+.../open-metadata/admin-services/users/{adminUserId}/servers/{serverName}/...
+```
+
+The **serverName** specified on these calls determines which configuration
+document is used, and hence which of the logical OMAG server's configuration it is working with.
+
+The OMAG server platform starts up without any open metadata capabilities enabled.
+Once it is running, it can be used to set up the configuration documents
+that describe the open metadata capabilities needed for each server instance.
+
+Once the configuration document is in place, the open metadata services
+can be activated and deactivated multiple times, across multiple
+restarts of the server platform.
+
+## Building a configuration document for a server
+
+The configuration document for the logical OMAG Server determines the types of open
+metadata and governance services that should be activated in the logical OMAG server.
+For example:
+
+* Basic descriptive properties of the server that are used in logging and events
+originating from the server.
+* What type of local repository to use.
+* Whether the Open Metadata Access Services (OMASs) should be started.
+* Which cohorts to connect to.
+
+Each of the configuration commands builds up sections in the configuration document.
+This document is stored in the configuration file after each configuration request so
+it is immediately available for use each time the open metadata services are activated
+in the OMAG Server.
+
+In the descriptions of the configuration commands the following values are used as examples:
+
+* The OMAG server platform is running on the localhost, at port 8080 (ie **http://localhost:8080**).
+* The user id of the administrator is **garygeeke**.
+* The name of the logical OMAG server (serverName) is **cocoMDS1**.
+
+### Common Configuration Tasks
+
+* [Setting basic properties for a logical OMAG server](configuring-ui-server-basic-properties.md)
```

Comment:
these tasks should be made UI specific

Reply:
The servername still needs updating this file as refers to cocoMDS1 - but am ok if you cover that in your followon doc issue? Feel free to close once noted?

Reply:
I see #2255 is open. will note there and close